config/badger_config.py

```diff
@@ -9,197 +9,41 @@
 
 sett_config = DotMap(
     native=DotMap(
-        convexRenCrv=DotMap(
-            strategyName="StrategyConvexStakingOptimizer",
+        convexMimCrv=DotMap(
+            strategyName="StrategyConvexStables",
             params=DotMap(
-                want=pools.renCrv.token,
-                pid=curve.pids.renCrv,
-                lpComponent=registry.tokens.wbtc,
+                want=pools.mimCrv.token,
+                pid=curve.pids.mimCrv,
                 performanceFeeStrategist=0,
                 performanceFeeGovernance=2000,
```

Comment:
To confirm, the performance fee would just be 20% gov perf fee

Reply:
The fee amount is yet to be confirmed by Po but 20% is what we have on the rest of the Convex vaults so it would make sense. 

I terms of whether it should be directed to the strategist or governance rewards we can do either. The idea would be to have these fees sent to a multisig to process them into BADGER. I can think of two options for this:
- Set the strategist to this multi and point all the fees to it.
- Setup a dedicated controller for these vaults and have their `controllers.rewards()` point to the multi so that governance fees go there.

Either of these two will do the trick, will talk to Dapp and @GalloDaSballo about which to pursue.

Reply:
Makes sense. Strategist would probably be the more straightforward way but will leave it to you